backend/emails/emails.js

```diff
@@ -15,7 +15,7 @@ const currencyFormatter = new Intl.NumberFormat('en-CA', {
 })
 
 const getEmailToSection = async (reporter_id, recipients) => {
-    const emailToSet = new Set(['jw4he@watonomous.ca', 'v2zheng@watonomous.ca'])
```

Comment:
Probably just make this an environment variable at this point, have it be a comma delimited list of default emails. or just have all 3 of us here